test/appium/views/home_view.py

```diff
@@ -174,3 +179,7 @@ def open_status_test_dapp(self, allow_all=True):
             else:
                 status_test_dapp.deny_button.click()
         return status_test_dapp
+
+    def check_empty_home_view(self):
```

Comment:
no need to create this method, you should search that chat name is not display, since empty home screen label might change

Reply:
In this case, I guess we need a separate small test to check the empty view, because it can break from time to time (I faced with this twice, when this message is not presented). 
That's why I added separate method here - it can be helpful IMO. If you think that it is unnecessary I can delete it.

Reply:
We should rely on values which are created by us or cannot be changed in future when using search by text (e.g. timestamps in amounts for transaction, public keys, `3 words user names` etc.)

Searching product captions by text may lead to endless changing of expected value and flaky test

Better ways are:

1. Search for no ChatElements are present at all (if home.chat_name_text.find_elements: fail 'Home screen is not empty').
2. Add accessibility label for empty chat caption in product and get element by label instead of text